Documentation/ceph-monitoring.md

```diff
@@ -81,20 +81,22 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
+### To enable prometheus alerts
+First, create the RBAC rules to enable monitoring,
+```BASH
+kubectl create -f cluster/examples/kubernetes/ceph/monitoring/monitoring-rbac.yaml
+``` 
+Then, make following changes to `cluster.yaml` and deploy.
 ```YAML
 monitoring:
   enabled: true
   rulesNamespace: "rook-ceph"
 ```
-_Note: This expects prometheus to be pre-installed by the admin._
 
-To enable prometheus alerts after cluster is deployed, run the following commands:  
 ```BASH
-cd cluster/examples/kubernetes/ceph/monitoring
-kubectl create -f prometheus-ceph-v<VERSION>-rules.yaml
```

Comment:
Do these rules not need to be applied at all?

Reply:
No. This is what Rook will take care of, based on the new fields added.

Reply:
Okay. I feel like I'm missing something. If this file is not used, why does it exist at all?

Reply:
With #3322 the operator will actually install the correct version of the rules automatically, so there is no need for this documented step to create the rules separately. This step should have been removed from #3322 before it was merged, but it was missed after we determined the final approach.

Reply:
So should that file be deleted here?

Reply:
No, the file will be automatically configured by the operator based on the CR setting for monitoring. These instructions just needed to be cleaned up.

Reply:
Okay. I see. The file has been deleted in master now.